examples/src/main/res/drawable/bottom_button_background.xml

```diff
@@ -2,7 +2,7 @@
 <selector xmlns:android="http://schemas.android.com/apk/res/android">
     <item android:state_enabled="true">
         <shape android:shape="rectangle">
-            <solid android:color="?attr/colorPrimary" />
+            <solid android:color="@color/colorPrimary" />
```

Comment:
after fixing the okhttp dependency conflict issue, this also crashes the InstructionViewActivity because it uses the `attr` in drawable before Lollipop. It's just an example, so use a fixed color to fix the issue.